forge/routes/api/deviceLive.js

```diff
@@ -296,14 +296,25 @@ module.exports = async function (app) {
             enabled: app.config.assistant?.enabled || false,
             requestTimeout: app.config.assistant?.requestTimeout || 60000,
             mcp: { enabled: true }, // default to enabled
-            completions: { enabled: true } // default to enabled
+            completions: { enabled: true, inlineEnabled: false } // default to enabled
         }
         if (app.config.assistant?.mcp && typeof app.config.assistant.mcp === 'object') {
             response.assistant.mcp = { ...app.config.assistant.mcp }
         }
         if (app.config.assistant?.completions && typeof app.config.assistant.completions === 'object') {
             response.assistant.completions = { ...app.config.assistant.completions }
         }
+        const isLicensed = app.license.active() || false
+        const licenseType = isLicensed ? (app.license.get('dev') ? 'DEV' : 'EE') : 'CE'
+        const tier = isLicensed ? app.license.get('tier') : null
+        const completionsTiers = [
+            'starter',
+            'teams', // AKA "pro" tier
+            'enterprise'
+        ]
```

Comment:
We have never tested for license 'type' before.

Also why do we need to test for team type? it should just test if the correct flag is present in the team, not what we (or any other enterprise licensed user has chosen to name their team)

Reply:
Added feature flag in [Enable Inline Completions via feature flag](https://github.com/FlowFuse/flowfuse/pull/5979/commits/c553e2bdf6abd77d421bf5603b0044f3e041eeae)